src/main/webapp/main-page.html

```diff
@@ -3,13 +3,37 @@
   <head>
     <meta charset="UTF-8">
     <title>Main Page</title>
+<<<<<<< HEAD
+    <link rel="stylesheet" href="style.css">
+=======
+>>>>>>> 746cdf1ef08b56098ec984c4ccd80a5e865b063d
```

Comment:
I think you accidentally pasted your git token in the code here. (It happened to me once too)

Reply:
Thanks for finding this, just fixed it.